impl/otbn/modexp_var.i.vad

```diff
@@ -483,6 +485,15 @@ procedure modexp_var_3072_f4(
 
     out_iter := b256_iter_cons(x21, 0, heap[x21].b256);
     out_iter := write_to_dmem(out_iter);
+
+    comment("/* If the counter value doesn't match expectations, cause a deliberate");
+    comment("   error (WDR reference > 31) to end the program. */");
+    li(x2, 232);
+    if (x2 != x5) {
+      li(x2, 255);
+      ghost var iter := b256_iter_cons(x21, 0, heap[x21].b256);
+      let next_iter := bn_sid_safe(x0, false, 0, x2, false, iter);
```

Comment:
I think it might be better to add an vale procedure that indicates error condition. So something like: 
```
procedure throw_error()
    requires false;
{
      li(x2, 255);
      ghost var iter := b256_iter_cons(x21, 0, heap[x21].b256);
      let next_iter := bn_sid_safe(x0, false, 0, x2, false, iter);
}
```
It should probably explicitly requires false to indicate abnormal behavior.

Reply:
Makes sense, fixed!